distributed/scheduler.py

```diff
@@ -410,16 +410,19 @@ def get_versions(self, comm):
         """ Basic information about ourselves and our cluster """
         return get_versions()
 
-    def start_services(self, listen_ip=''):
+    def start_services(self, host):
         for k, v in self.service_specs.items():
             if isinstance(k, tuple):
                 k, port = k
             else:
                 port = 0
 
+            if host == '0.0.0.0':
+                host = ''  # for IPv6
```

Comment:
I see. Where does "0.0.0.0" come from? Something must have changed which made that value appear...